packages/paypal-commerce-integration/src/paypal-commerce-request-sender.ts

```diff
@@ -32,9 +33,9 @@ export default class PayPalCommerceRequestSender {
 
         return res.body;
     }
-
-    // TODO: add response type interface Promise<something> in another task
-    async updateOrder(requestBody: PayPalUpdateOrderRequestBody) {
+    async updateOrder(
```

Comment:
May I ask you to add an empty line between `createOrder` and `updateOrder` methods?